src/Weasel.Postgresql/Tables/IndexDefinition.cs

```diff
@@ -63,6 +63,8 @@ protected virtual string deriveIndexName()
 
         public SortOrder SortOrder { get; set; } = SortOrder.Asc;
 
+        public NullsSortOrder? NullsSortOrder { get; set; } = null;
```

Comment:
This was on me here, but we could use some Xml comments. I'm happy to help w/ that after this PR.

Reply:
Xml comments added.